pkg/kubelet/cm/node_container_manager_linux.go

```diff
@@ -135,10 +135,8 @@ func enforceExistingCgroup(cgroupManager CgroupManager, cName CgroupName, rl v1.
 	if !cgroupManager.Exists(cgroupConfig.Name) {
 		return fmt.Errorf("%q cgroup does not exist", cgroupConfig.Name)
 	}
-	if err := cgroupManager.Update(cgroupConfig); err != nil {
-		return err
-	}
-	return nil
+	err := cgroupManager.Update(cgroupConfig)
```

Comment:
missed this one :-)

Reply:
Thanks for the review. updated  now PTAL